### ISSUE TITLE
Chest pain described as ripping or tearing

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10136]
**Preferred term label:**
Chest pain described as ripping or tearing

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://www.ncbi.nlm.nih.gov/medgen/C5238909

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Aortic dissection

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000049
- label: Chest pain described as ripping or tearing
- parents: [Pain characteristic (HP:0025280)](https://hpo.jax.org/app/browse/term/HP:0025280)
- pmid: PMID:21250256
- def: Chest pain described as a tearing or ripping sensation.
- comment: The pain of aortic dissection may be described as tearing or ripping and does not improve with nitroglycerin.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10136
- synonyms:
